models/models.go

```diff
@@ -15,8 +15,8 @@ type JobResults struct {
 
 // Job for importing datasets
 type Job struct {
-	JobID         string          `bson:"id,omitempty"             json:"id,omitempty"`
-	Recipe        string          `bson:"recipe,omitempty"         json:"recipe,omitempty"`
+	ID            string          `bson:"id,omitempty"             json:"id,omitempty"`
+	RecipeURL     string          `bson:"recipe,omitempty"         json:"recipe,omitempty"`
```

Comment:
Should this create the following JSON structure { "links": { "recipe": { "id":"123-456", "href":"http://........." }}}?

Reply:
Good point, it seems like a reasonable suggestion - will discuss this further before changing.

Reply:
Item added to tech debt log to be picked up when we implement the recipe API.